### PR TITLE
IL : Add i24 News (Arabic)

### LINKED ISSUE #2986: i24NEWS Arabic
CAN ANY ONE EXTRACT THE M3U8 LINK OF THIS LIVE STREAM 
https://www.i24news.tv/ar/%D9%85%D8%A8%D8%A7%D8%B4%D8%B1

### PR DESCRIPTION
**Pull request by BellezaEmporium**

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Put on draft, weird issue when getting the link, it simply does not seem to work properly.

**Comment by MapGuy11** [comment]
Link works for me in the US.

**Comment by LaneSh4d0w** [comment]
It works for you ? Interesting... my VLC literally stops when he gets the playlist.

**Comment by LaneSh4d0w** [comment]
> Also btw "Wowzers Streaming Service version Pog 420.69 -> Shade edition" lol :)

Haha, a little reference to Wowza's services x)

**Comment by MapGuy11** [comment]
> It works for you ? Interesting... my VLC literally stops when he gets the playlist.

Not on VLC, but on my web browser, it redirects correctly.

**Comment by themak1985** [comment]
Yeah, link does not work in VLC for me - I don't get an error - just nothing happens.

**Comment by LaneSh4d0w** [comment]
Yeah this is going to be a bit more difficult than expected.

**Comment by tilfizyawn** [comment]
@LaneSh4d0w @freearhey 

The link works on Android Chrome, maybe it should be marked as i24 News (Arabic) [Mobile only]

and we should add that exclusion to https://github.com/iptv-org/iptv/blob/master/scripts/clean.js#L31

**Comment by LaneSh4d0w** [comment]
Interesting...

**Comment by freearhey** [comment]
@tilfizyawn the link also works on desktop
image

**Comment by LaneSh4d0w** [comment]
Works on my browser, indeed, but not on my main Android IPTV app.

**Comment by LaneSh4d0w** [comment]
Reopening since there's some positive results being shown.

File: channels/il.m3u [hunk 1]
```diff
@@ -27,6 +27,8 @@ https://stream71.shidur.net/htvlive2/_definst_/smil:live2.smil/playlist.m3u8
 https://stream72.shidur.net/htvlive2/_definst_/smil:live2.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="IsraelParsTV.il" tvg-name="Israel Pars TV" tvg-country="IL" tvg-language="" tvg-logo="" group-title="",Israel Pars TV
 https://live.pars-israel.com/iptv/stream.m3u8
+#EXTINF:-1 tvg-id="i24NewsArabic.il" tvg-name="i24 News (Arabic)" tvg-country="IL" tvg-language="Arabic" tvg-logo="" group-title="News",i24 News (Arabic)
+https://iptv-all.lanesh4d0w.repl.co/israel/i24arabic
```
Comment (freearhey) [inline channels/il.m3u:31]: The link still does not work via VLC.
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/il.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/il.m3u:31]: Yes, this is apparently due to some things on their side, mine simply redirects to the resource.

Comment (Dum4G) [reply channels/il.m3u:31]: Do you also apply headers like referer and origin? Have you tried different user agents?

Comment (LaneSh4d0w) [reply channels/il.m3u:31]: Nothing like that, it's straight-forward. It is a link generated by their servers that I simply parse.

**Comment by Dum4G** [comment]
Whenever I open this channel it broadcasts in English. I don't think we should specify a language for this one

**Comment by LaneSh4d0w** [comment]
> Whenever I open this channel it broadcasts in English. I don't think we should specify a language for this one

It's actually a mix of both languages. English at one time, Arabic on another.

**Comment by freearhey** [comment]
@LaneSh4d0w Can I close this PR?

**Comment by LaneSh4d0w** [comment]
You can, I think I won't get much results from it. Don't seem to have a workaround for now.

**Pull request closed without merging**